src/oracles.jl

```diff
@@ -303,3 +303,61 @@ function compute_extreme_point(
     end
     return storage
 end
+
+"""
+"""
+struct ChasingGradientLMO{LMO<:LinearMinimizationOracle,T} <: LinearMinimizationOracle
+    inner::LMO
+    max_rounds_number::Int
+    improv_tol::T
+end
+
+"""
+"""
```

Comment:
No need for a docstring on the method, it should just be explained on the docstring of the ChasingGradientLMO